google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStandardRunTask.java

```diff
@@ -21,18 +21,28 @@
 import com.google.cloud.tools.appengine.cloudsdk.CloudSdkAppEngineDevServer;
 import com.google.cloud.tools.appengine.cloudsdk.process.ProcessStartListener;
 import com.google.cloud.tools.intellij.appengine.sdk.CloudSdkService;
+import com.google.cloud.tools.intellij.stats.UsageTrackerProvider;
+import com.google.cloud.tools.intellij.util.GctTracking;
+import com.google.common.base.Strings;
 
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 /**
  * Represents an App Engine Standard run task. (i.e., devappserver)
  */
 public class AppEngineStandardRunTask extends AppEngineTask {
 
   private RunConfiguration runConfig;
+  private String runnerId;
 
-  public AppEngineStandardRunTask(@NotNull RunConfiguration runConfig) {
+  /**
```

Comment:
I followed Etan's suggestion to remove the unnecessary descriptive comment, but in that case I'll re-add it, because I want to specify what runnerId is.